apps/projects/app/components/Content/IssueDetail/components/Detail.js

```diff
@@ -0,0 +1,247 @@
+  return <StyledTable>{mappedTableFields}</StyledTable>
+}
+
+// this 10px padding and...
+const Wrapper = styled.div`
+  display: flex;
+  height: 100%;
+  padding: 10px;
+`
+// todo: wrapper component for different sizes (changes padding mostly)
+
+// ...that 10px margin result in a 20px gap
+const cardStyle = {
+  flex: '0 1 auto',
+  textAlign: 'left',
+  padding: '15px 30px',
+  margin: '10px',
+  background: theme.contentBackground,
+  border: `1px solid ${theme.contentBorder}`,
+  borderRadius: '3px',
+}
+
+const column = {
+  display: 'flex',
+  flexDirection: 'column',
+  flexBasis: '100%',
+}
+
+// TODO: Remove fake default value for img
+const Avatar = ({ size, img = githubLogo }) => {
+  // do something with the size...
+  const avatarStyle = () => {
+    switch (size) {
+    case 'small':
+      return { transform: 'scale(.6)' }
+    default:
+      return { transform: 'scale(.8)' }
+    }
+  }
+
+  return (
+    <div>
+      <img src={img} alt="user avatar" style={avatarStyle()} />
+    </div>
+  )
+}
+
+const MemberRow = ({ name, role, status, avatar }) => (
+  <Wrapper>
+    <Avatar size="normal" style={column}>
+      {avatar}
+    </Avatar>
+    <div style={{ ...column, flex: 2 }}>
+      <Text.Block>{name}</Text.Block>
+      <Text.Block>{role}</Text.Block>
+      <Text.Block>{status}</Text.Block>
+    </div>
+    <ContextMenu>
+      <ContextMenuItem>In progress...</ContextMenuItem>
+    </ContextMenu>
+  </Wrapper>
+)
+const ActivityRow = ({ name, log, date, avatar }) => (
+  <Wrapper style={{ padding: '0' }}>
+    <Avatar size="small" style={column}>
+      {avatar}
+    </Avatar>
+    <div style={{ ...column, flex: 2 }}>
+      <Text.Block>
+        {name} {log}
+      </Text.Block>
+      <Text.Block>{date}</Text.Block>
+    </div>
+  </Wrapper>
+)
+
+const fakeActivities = [
+  {
+    name: 'Worf',
+    log: 'began the task',
+    date: '2 days ago',
+    avatar: null,
+  },
+  {
+    name: 'Tasha Yar',
+    log: 'assigned Worf',
+    date: '3 days ago',
+    avatar: null,
+  },
+  {
+    name: 'Data',
+    log: 'rejected Jean-Luc\'s work',
+    date: 'Last seen 4 hours ago',
+    avatar: null,
+  },
+]
+
+const fakeMembers = [
+  {
+    name: 'Worf',
+    role: 'Contributor',
+    status: 'Pending assignment',
+    avatar: null,
+  },
+  {
+    name: 'Tasha Yar',
+    role: 'Contributor',
+    status: 'Assignment approved',
+    avatar: null,
+  },
+  {
+    name: 'Data',
+    role: 'Task Manager',
+    status: 'Last seen 4 hours ago',
+    avatar: null,
+  },
+]
+
+const Detail = ({
+  bountyBadge = '100 ANT',
+  title,
+  number,
+  repo,
+  body,
+  activities = fakeActivities, // TODO: Remove default fake value when data arrives from backend
+  team = fakeMembers, // TODO: Also this
+  summaryData, // an object containing {experience, deadline, slots, status} for that issue
+}) => {
+  const calculatedDate = () => {
+    // something obtained from props and transformed to be like:
+    return 'Opened 4 days ago'
```

Comment:
This might be doable straight away?

Reply:
Solved at: 92436441ff015d9a1fc9eaf9bc57db71571e8e85